prow/plugins/assign/assign.go

```diff
@@ -188,7 +188,7 @@ type handler struct {
 func newAssignHandler(e github.GenericCommentEvent, gc githubClient, log *logrus.Entry) *handler {
 	org := e.Repo.Owner.Login
 	addFailureResponse := func(mu github.MissingUsers) string {
-		return fmt.Sprintf("GitHub didn't allow me to assign the following users: %s.\n\nNote that only [%s members](https://github.com/orgs/%s/people) and repo collaborators can be assigned. \nFor more information please see [the contributor guide](https://git.k8s.io/community/contributors/guide/#issue-assignment-in-github)", strings.Join(mu.Users, ", "), org, org)
+		return fmt.Sprintf("GitHub didn't allow me to assign the following users: %s.\n\nNote that only [%s members](https://github.com/orgs/%s/people) and repo collaborators can be assigned and that issues/PRs can only have 10 assignees at the same time.\nFor more information please see [the contributor guide](https://git.k8s.io/community/contributors/guide/#issue-assignment-in-github)", strings.Join(mu.Users, ", "), org, org)
```

Comment:
Maybe

```
Note that only ten or fewer [%s members...
```

?

Reply:
(this is a nit-pick)

Reply:
Can't we also detect the >10 case here and make the error message specific to the error that happened?

Reply:
Assigning 11+ people at the same time seems like an edge case. How do you feel about only mentioning this fact if a) more than 10 people were attempted to be assigned and b) github returns an error?

Reply:
Actually given @ibrasho comments above that isn't going to work.